scripts/build.py

```diff
@@ -609,6 +607,51 @@ def build_files(source, target, file_hashes):
             print e
 
 
+def rebuild_new_files(source_path, target_path, new_files_list, file_hashes):
+    """Minifies list of  new files, removes whitespace from HTML and
+    interpolates paths in HTML to include hashes in source
+    directory and copies it to target.
+
+    Args:
+        source_path: str. Path relative to /oppia directory of directory
+            containing files and directories to be copied.
+        target_path: str. Path relative to /oppia directory of directory where
+            to copy the files and directories.
+        new_files_list: list. List of recently changed file.
```

Comment:
Specify the type of the elements in the list, e.g. list(int) or list(str) or ...

At end of docstring: file --> files

Reply:
Updated docstring accordingly.